docker.go

```diff
@@ -664,6 +664,10 @@ func WithDefaultBridgeNetwork(bridgeNetworkName string) DockerProviderOption {
 
 func NewDockerClient() (cli *client.Client, host string, tcConfig TestContainersConfig, err error) {
 	tcConfig = readTCPropsFile()
```

Comment:
Wdyt about renaming the `readTCPPropsFile` method to `configureTC` and perform all operations in there: read the Props File and the env vars?